codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -52,26 +55,43 @@
         extends MessageAggregator<HttpObject, HttpMessage, HttpContent, FullHttpMessage> {
 
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(HttpObjectAggregator.class);
-    private static final FullHttpResponse CONTINUE = new DefaultFullHttpResponse(
-            HttpVersion.HTTP_1_1, HttpResponseStatus.CONTINUE, Unpooled.EMPTY_BUFFER);
+    public static final int DEFAULT_MAX_COMPOSITEBUFFER_COMPONENTS = 1024;
```

Comment:
@normanmaurer - Good catch. Missed this in forward port...this now lives in `MessageAggregator`